apps/catalogue/src/store/mutations.js

```diff
@@ -1,6 +1,6 @@
 export default {
   setVariables(state, variables) {
-    Vstate.variables = Array.isArray(variables) ? variables : [];
+    state.variables = Array.isArray(variables) ? variables : [];
```

Comment:
nice . but this is part of separate PR waiting for review , having a single action ( feat, fix, chore ) in each PR makes it easer to release and maintain the app

Reply:
yes, would be happy if the other is merged first !